DiscordChatExporter.Domain/Exporting/ExportContext.cs

```diff
@@ -45,13 +45,13 @@ internal class ExportContext
             var dateFormat => date.ToLocalString(dateFormat)
         };
 
-        public Member? TryGetMember(Snowflake id) => Members.FirstOrDefault(m => m.Id == id);
+        public Member? TryGetMember(Snowflake? id) => Members.FirstOrDefault(m => m.Id == id);
```

Comment:
Instead of making `Snowflake` parameter nullable here, can we change the callsite (in `HtmlMarkdownVisitor.cs` etc) to something like this:

```csharp
var mentionId = Snowflake.TryParse(mention.Id);
var member = mentionId?.Pipe(_context.TryGetMember);
```

Member will then either be `null` or resolved.